client/man/ipa-getkeytab.1

```diff
@@ -88,11 +88,7 @@ This options returns a description of the permitted encryption types, like this:
 Supported encryption types:
 AES\-256 CTS mode with 96\-bit SHA\-1 HMAC
 AES\-128 CTS mode with 96\-bit SHA\-1 HMAC
-Triple DES cbc mode with HMAC/sha1
```

Comment:
Add AES 128/256 SHA-2 here as well?

AES-128 CTS mode with 128-bit SHA-256 HMAC
AES-256 CTS mode with 192-bit SHA-384 HMAC

Reply:
Ah, yes, thank you.